From 32ace708bdaa5bf0494274661aacc5beff527cdc Mon Sep 17 00:00:00 2001
From: Martin Klapetek <mklapetek@kde.org>
Date: Wed, 6 Jan 2016 22:59:14 -0500
Subject: [PATCH 1/6] [notifications] Ensure the screen position selector works
 after state change

Disabling it and enabling back again gives it stale data. This ensures
the selector is properly cleared when disabled.
---
 applets/notifications/package/contents/ui/ScreenPositionSelector.qml | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/applets/notifications/package/contents/ui/ScreenPositionSelector.qml b/applets/notifications/package/contents/ui/ScreenPositionSelector.qml
index efff648..4b848c5 100644
--- a/applets/notifications/package/contents/ui/ScreenPositionSelector.qml
+++ b/applets/notifications/package/contents/ui/ScreenPositionSelector.qml
@@ -39,8 +39,10 @@ QtControls.GroupBox {
 
     onEnabledChanged: {
         if (!enabled) {
-            selectedPosition = NotificationsHelper.Default
+            positionRadios.current = null
         }
+
+        selectedPosition = NotificationsHelper.Default
     }
 
     PlasmaCore.Svg {
-- 
2.5.0

